uncontrast_studies/processors/parameters_distribution_bar.py

```diff
@@ -0,0 +1,183 @@
+import itertools
+
+from django.contrib.postgres.expressions import ArraySubquery
+from django.db.models import QuerySet, OuterRef, F, Func, Count
+from django.db.models.functions import JSONObject
+
+from uncontrast_studies.processors.base import BaseProcessor
+from uncontrast_studies.models import (
+    UnConMainParadigm,
+    UnConExperiment,
+    UnConSample,
+    UnConTaskType,
+    UnConModalityType,
+    UnConStimulusCategory,
+    UnConProcessingMainDomain,
+    UnConsciousnessMeasureType,
+    UnConsciousnessMeasurePhase,
+)
+
+
+class ParametersDistributionBarGraphDataProcessor(BaseProcessor):
+    def __init__(self, experiments: QuerySet[UnConExperiment], **kwargs):
+        super().__init__(experiments=experiments, **kwargs)
+        breakdown = kwargs.pop("breakdown")
+        self.breakdown = breakdown[0]
+        # Just so i can copy paste the methods
+        self.filtered_experiments = self.experiments
+
+    def process(self):
+        process_func = getattr(self, f"process_{self.breakdown}")
+        return process_func()
+
+    def process_paradigm(self):
+        experiments_subquery_by_breakdown = self.filtered_experiments.filter(paradigm__main=OuterRef("pk")).values(
+            "id"
+        )  # TODO: adapt this one according to the final modeling
+
+        breakdown_query = UnConMainParadigm.objects.values("name").distinct().annotate(series_name=F("name"))
+
+        qs = self._aggregate_query_by_breakdown(breakdown_query, experiments_subquery_by_breakdown)
+        return qs
+
+    def process_population(self):
```

Comment:
I prefer naming it "process_sample_type" to be more consistent with the model name

Reply:
I know, but it's from the "original" system, and in this point we're stuck with it